tools/run_danesfield.py

```diff
@@ -224,23 +227,46 @@ def main(args):
 
     p3d_file = config['paths']['p3d_fpath']
 
+    #############################################
+    # Run VisSat pipeline
+    #############################################
+    if args.vissat:
+        aoi_config = config['paths'].get('aoi_config')
+        if aoi_config == None:
+            print("Error: Path to aoi_config file must be provided when using VisSat")
+            exit(1)
+        with open(aoi_config) as f:
+            data = json.load(f)
+
+        vissat_workdir = data['work_dir']
+        cmd_args = ["python3", "/VisSatSatelliteStereo/stereo_pipeline.py", "--config_file", aoi_config]
+        run_step(vissat_workdir, "VisSat", cmd_args)
+        cmd_args = ["python3", "/ply2txt.py", os.path.join(vissat_workdir, 'mvs_results/aggregate_3d/aggregate_3d.ply'), os.path.join(vissat_workdir, 'mvs_results/aggregate_3d/aggregate_3d.txt')]
```

Comment:
It would be good to put line breaks in some of these long lines.  We should generally follow PEP8 style for Python so we want to break lines longer than 79 characters when possible.